mysql-test/std_data/keyring_vault_confs/keyring_vault.conf

```diff
@@ -0,0 +1,4 @@
+vault_url = https://127.0.0.1:8200
+secret_mount_point = secret
+token = 8b1f38f6-cbe3-fcaa-ba22-7196c792e91a
+vault_ca = /home/rob/vault_certs/vault_ca.crt
```

Comment:
Path to CA certificate must be absolute - this is how CURL requires it. It will need to be set separately for each server. That is why I left it as that ...